src/pages/[platform]/reference/project-structure/index.mdx

```diff
@@ -28,7 +28,7 @@ export function getStaticProps(context) {
   };
 }
 
-Amplify Gen 2 backends are defined using TypeScript, and enable you to collocate resources depending on their function. For example, you can author a [post confirmation trigger for Amazon Cognito](https://docs.aws.amazon.com/cognito/latest/developerguide/user-pool-lambda-post-confirmation.html) right next to your auth's resource file.
+Amplify Gen 2 backends are defined using TypeScript, and enable you to collocate resources depending on their function. For example, you can author a [post confirmation trigger for Amazon Cognito](/[platform]/build-a-backend/functions/examples/create-user-profile-record/) right next to your auth's resource file.
```

Comment:
what is the motivation for this change?

Reply:
pointing to an example on how to create this use case